TestProject/output/GIFOutput.php

```diff
@@ -55,7 +55,13 @@ function finishOutput()
         try {
             //An animated gif is automatically created and saved under the respective repository.
             $gif->create(glob("imageOutput/*.png"), 50, 0);
-            file_put_contents("imageOutput/animation.gif", $gif->getGif());
+            file_put_contents("imageOutput/gifAnimation.gif", $gif->getGif());
+
+            //Deletes all current images
+            foreach (glob("imageOutput/pngImage.png") as $filename)
```

Comment:
Fix deleting only the gif-output-specific images. (You should use the same glob-expression when you search for the individual images for the gif-creation.)